benchmarks/tensorexpr/benchmark.py

```diff
@@ -82,6 +85,20 @@ def compute_workload(self):
         """return the number of scalar operations it takes to finish the tensor op"""
         return None
 
+    @staticmethod
+    def input_iterable():
+        """A benchmark child class should return true if it utilizes the input iter arg"""
+        return False
+
+    def dtype_to_bytes(self) :
+        import torch
+        if self.dtype == torch.float32 :
```

Comment:
You could ask torch for the size of a dtype by doing `torch.tensor(0, dtype=self.dtype).element_size()`.  I have mixed feelings about creating a tensor just for this purpose, but it does at least keep the source of truth in `torch` instead of our benchmark harness :-p.

Reply:
I am perfectly fine with this.  I just added the method as I wasn't finding an ideal choice.  One other note is that I think the Dtype based memory calculations will need to change again.  I was about to add some ops that had mixed precision and quickly realized that generically multiplying by a type size was not going to work.